examples/capacity_factor_example.py

```diff
@@ -35,13 +36,51 @@
         ]
     )
 
+    # This is a real capacity time series, which must be normalized
```

Comment:
This is fine, but I think we should maybe turn off formatting for example files? @GabrielSoto-INL what do you think?